guizero/LabelBox.py

```diff
@@ -0,0 +1,114 @@
+from tkinter import LabelFrame
+from . import utilities as utils
+from .base import ContainerTextWidget
+from .event import EventManager
+
+class LabelBox(ContainerTextWidget):
+
+    def __init__(
+        self,
+        master,
+        text="",
+        layout="auto",
+        grid=None,
+        align=None,
+        visible=True,
+        enabled=None,
+        width=None,
+        height=None,
+        border=None):
```

Comment:
if `border` is None, does the LabelBox have a border by default? That seems to be the default for tkinter.LabelFrame but I havent checked.

Reply:
The default behavior is that it has a frame around the frame's content. The setting border to anything different than 0 or None will draw a line of varying thickness around the widget AND the label effectively showing two borders. The attached image makes it clearer
![labelbox-border](https://user-images.githubusercontent.com/1418331/140579574-40b2db79-9ca8-413a-a2aa-da4f2f38852c.png)

Therefore, I will add additional parameters to control the frame